packages/extras/src/lib/components/Text3D/Text3D.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+  import {
+    T,
+    forwardEventHandlers,
+    useLoader,
+    type AsyncWritable,
+    asyncWritable
+  } from '@threlte/core'
+  import { createEventDispatcher } from 'svelte'
+
+  import type { Mesh } from 'three'
+  import { TextGeometry, FontLoader, Font } from 'three/examples/jsm/Addons.js'
```

Comment:
We should use individual import paths here due to this issue: https://github.com/threlte/threlte/pull/871